### ISSUE TITLE
Cannot generate line.thrift

### ISSUE THREAD

**Issue by nazri07** [BroNils/LINE-Source#5]
Please update fix line protocol :)

**Comment by BroNils** [comment]
From where you download it ? (give me name of github folder & file) [ex: LINE-Source/old (buggy)/line.thrift]

**Comment by BroNils** [comment]
For now I still don't have time to maintain this project, maybe later i'll fix.

**Comment by nazri07** [comment]
yes LINE-Source/old (buggy)/line.thrift , please update :) i'm very grateful
